<?xml version="1.0"?>
<launch>

<node pkg="nathan" type="SearchService.py" name="search_server" output="screen"/>
<node pkg="nathan" type="ProbeSurfaceService.py" name="probe_surface_server" output="screen"/>

</launch>